parser/parser.h

```diff
@@ -2,6 +2,7 @@
 #define SORBET_PARSER_PARSER_H
 
 #include "Node.h"
+#include "ResolvedConst.h"
```

Comment:
This should be obviated by moving the ResolvedConstant node into generate_ast.cc